tools/ALARAJOYWrapper/alarajoy_QA.py

```diff
@@ -207,4 +208,428 @@ def process_data(
             variable, unit = variable_w_unit.split(' [')
             dfs.update(make_entry(datalib, variable, unit.strip(']'), data))
 
-    return dfs
\ No newline at end of file
+    return dfs
+
+#------------------------- Plotting Helper Functions -------------------------
+
+def process_time_vals(alara_df, seconds=True):
+    '''
+    Convert the cooling times of the ALARA analysis post-shutdown to seconds.
+    Arguments:
+        alara_df (pandas.core.frame.DataFrame): DataFrame containing the
+            extracted tabular data for a single variable and interval/zone of
+            an ALARA run.
+        seconds (bool, optional): Option to convert cooling times from years
+            to seconds.
+            (Defaults to True)
+    Returns:
+        times (list): List of the ALARA cooling times, converted to seconds
+    '''
+
+    times = []
+    time_dict = {'shutdown' : 0.0}
+    time_dict['y'] = 365*24*60*60 if seconds else 1
+
+    for column in alara_df.columns[1:]:
+        if column == 'shutdown':
+            times.append(time_dict['shutdown'])
+        else:
+            time = column.split('y')[0]
+            times.append(float(time) * time_dict['y'])
+
+    return times
+
+def separate_total(alara_df):
```

Comment:
I wonder if we should update `alara_output_parser` to always ignore the total?  We can always recreate it by summing over a dataframe, and then don't have to worry about separating it...